.github/workflows/typecheck.yml

```diff
@@ -8,16 +8,16 @@ jobs:
     strategy:
       matrix:
         node: ["14"]
-    name: Flow check
+    name: TypeScript check
     steps:
       - uses: actions/checkout@v2
       - name: Setup node ${{ matrix.node }}
-        uses: actions/setup-node@v2.5.0
+        uses: actions/setup-node@v2.4.0
```

Comment:
why did we downgrade the version here?

Reply:
This seems to be a mistake and should be v2.5.0.